src/lib/platform/EiScreen.h

```diff
@@ -107,6 +109,19 @@ class EiScreen : public PlatformScreen {
     void on_motion_event(ei_event *event);
     void on_abs_motion_event(ei_event *event);
 
+    void handle_ei_log_event(struct ei *ei,
```

Comment:
In C++ code struct is not needed. The compiler already knows that `ei` is a struct name due to the forwards declaration above. Also, align * with the type name.

Reply:
> In C++ code struct is not needed.

My puny C mind is well and truly blown! :) Thanks for the fixes and merges!